lib/internal/errors.js

```diff
@@ -837,6 +837,7 @@ E('ERR_INVALID_SYNC_FORK_INPUT',
   TypeError);
 E('ERR_INVALID_THIS', 'Value of "this" must be of type %s', TypeError);
 E('ERR_INVALID_TUPLE', '%s must be an iterable %s tuple', TypeError);
+E('ERR_INVALID_UNIX_ID', 'Unix %s id does not exist: %s', Error);
```

Comment:
`ERR_INVALID_CREDENTIAL`/`ERR_UNKNOWN_CREDENTIAL`? The error message is unambiguous, but it’s not really clear what’s meant by “Unix id” from just seeing the error code, imo

Reply:
What about `ERR_UNKNOWN_IDENTIFIER` ?

Reply:
And changing the message to "Group/User identifier does not exist: %s" ?

Reply:
@targos I think that might be a bit generic, especially since “unknown identifier” is a term that has a different meaning in a programming language context

I’m fine with changing the error message to what you’re suggesting, but I’m also fine with it the way it is right now :)

Reply:
`ERR_INVALID_CREDENTIAL` sounds better to me since you'd do `man credentials` to read the manual on this...or maybe `ERR_INVALID_PRIVILEGES`?

Reply:
Oh, I changed it to `ERR_INVALID_CREDENTIAL` but forgot to push. Done now.